src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -308,7 +308,7 @@ public void AsymmetricDSASignature()
 
             Assert.Null(signedXml.SignatureLength);
             Assert.Equal(SignedXml.XmlDsigDSAUrl, signedXml.SignatureMethod);
-            Assert.Equal(64, signedXml.SignatureValue.Length);
+            Assert.True(signedXml.SignatureValue.Length >= 40, $"At {signedXml.SignatureValue.Length} bytes long, the signature value is less than the minimum length of 40 bytes.");
```

Comment:
Odd, I thought I'd looked and there aren't any inequality asserts besides NotEqual. Will have a other look when I get back to computer :)